addons/survey/static/src/question_page/question_page_one2many_field.js

```diff
@@ -12,6 +15,92 @@ class QuestionPageOneToManyField extends X2ManyField {
         useSubEnv({
             openRecord: (record) => this.openRecord(record),
         });
+        this.notificationService = useService("notification");
+
+        // Systematically and automatically save SurveyForm at each question edit/creation mainly
+        // enables allows checking validation parameters consistency and use questions as triggers
+        // immediately during question creation.
+        // Preparing everything in order to override `this._openRecord` below.
+        const { saveRecord: superSaveRecord, updateRecord: superUpdateRecord } = useX2ManyCrud(
+            () => this.list,
+            this.isMany2Many
+        );
+
+        const self = this;
+        const saveRecord = async (record) => {
+            const saveResponse = await superSaveRecord(record);
+            try {
+                await self.props.record.save({stayInEdition: true, throwOnError: true});
+            } catch (error) {
+                return self.handleSurveySaveError(error, record);
+            }
+            return saveResponse;
+        };
+
+        const updateRecord = async (record) => {
+            const updateResponse = await superUpdateRecord(record);
+            try {
+                await self.props.record.save({stayInEdition: true, throwOnError: true});
+            } catch (error) {
+                return self.handleSurveySaveError(error);
+            }
+            return updateResponse;
+        };
+
+        const openRecord = useOpenX2ManyRecord({
+            resModel: this.list.resModel,
+            activeField: this.activeField,
+            activeActions: this.activeActions,
+            getList: () => this.list,
+            saveRecord,
+            updateRecord,
+        });
+        this._openRecord = async (params) => {
+            if (!await self.props.record.save({stayInEdition: true})) {
+                return;
+            }
+            const activeElement = document.activeElement;
+            if (params.record) {
+                // Trigger loading for values that may have changed when reordering/deleting
+                // rows (allowed_triggering_question_ids, is_conditional, is_placed_before_trigger).
+                await params.record.load();
+            }
+            openRecord({
+                ...params,
+                onClose: () => {
+                    if (activeElement) {
+                        activeElement.focus();
+                    }
+                },
+            });
+        };
+    }
+
+    /**
+     * As it is convenient to keep the questions modal open, two things should
+     * be cared for in case of error while saving the form after adding or
+     * updating a question:
+     *   * Remove erroneous question row added
+     *   * Replace default error modal with a notification
```

Comment:
In that case the modal form view of the question does NOT close, correct?
Could be nice to explain that we stay in question edition.